libcxx/test/libcxx/utilities/memory/util.smartptr/util.smartptr.shared/libcxx.control_block_layout.pass.cpp

```diff
@@ -145,15 +198,20 @@ int main(int, char**) {
   test<TrivialNonEmptyType, NonTrivialAlloc>();
 
   test<FinalEmptyType, TrivialEmptyAlloc>();
-  test<FinalEmptyType, TrivialNonEmptyAlloc>();
-  test<FinalEmptyType, FinalEmptyAlloc>();
-  test<FinalEmptyType, NonTrivialAlloc>();
+  // test<FinalEmptyType, TrivialNonEmptyAlloc>();
```

Comment:
Leftover? I would leave behind a comment that we know `FinalEmptyType` not to pass this test when the allocators are X, Y and Z.